umcontroller/umhandler.go

```diff
@@ -32,12 +32,18 @@ import (
 /***********************************************************************************************************************
  * Types
  **********************************************************************************************************************/
+type closeReason string
+
+const (
```

Comment:
Why not use enum instead of string constant?

Reply:
fixed